### ISSUE TITLE
Linking issues

### ISSUE THREAD

**Issue by gramian** [ArcadeData/arcadedb-docs#43]
Dear Developers,

I found some minor local linking issues with the documentation I list below, sorted by sections:

* **1. Jump to the Hot Topics**
  * [x] "[Document Database](https://docs.arcadedb.com/#_document-model)" does not work  
  * [x] "[Key/Value](https://docs.arcadedb.com/#_keyvalue-model)" does not work
  * [x] "[Time Series](https://docs.arcadedb.com/#_timeseries-model)" does not work
  * [x] "[Apache Gremlin (Apache Tinkerpop v3.4.x)](https://docs.arcadedb.com/#_gremlin-api)" does not work
  * [x] "[MongoDB Query Language](https://docs.arcadedb.com/#_mongodb-api)" does not work
  * [x] "[Embedded](https://docs.arcadedb.com/#_)" does not work
  * [x] "[HTTP/JSON](https://docs.arcadedb.com/#_http-api)" does not work
  * [x] "[Postgres driver](https://docs.arcadedb.com/#_postgres-driver)" does not work
  * [x] "[MongoDB driver](https://docs.arcadedb.com/#_mongodb-api)" does not work
  * [x] "[Redis driver](https://docs.arcadedb.com/#_redis-api)" does not work
  * [x] "[Java Tutorial](https://docs.arcadedb.com/#_java-tutorial)" does not work
* **2. Introduction**
  * [x] "2.2.2 Document Model": "`[types](Concepts.md#type)`" is md instead of asciidoctor syntax
  * [x] "2.2.2 Document Model": "[Link](https://docs.arcadedb.com/#Link)" does not work
* **3. Main Concepts**
  * [x] Why does this section not have sub-section numbers? 
  * [x] "Types": "[configurable strategy](https://docs.arcadedb.com/#_bucket-selection)" does not work
  * [x] "Transactions": "`[consistent](#consistency), [isolated](#isolation) and [durable](#durability)`" are md instead of asciidoctor syntax
  * [x] "Transactions": "[ACID](https://docs.arcadedb.com/#_-acid-properties)" do not work (twice)
  * [x] "Inheritance": Should "OrientDB" be here "ArcadeDB"?
  * [x] "Declaring Inheritance Schema": "`[POJO’s](../java/Object-DB-Interface.md#inheritance) and [Documents](../java/Document-Database.md)`" are md instead of asciidoctor syntax
  * [x] "Using Polymorphic Queries": Should "OrientDB" actually be here "ArcadeDB"? (twice)
* **4. Security**
  * [x] "4.1 Server": "[Security Policy](https://docs.arcadedb.com/#Security-Policy)" does not work
  * [x] "4.5 Settings": "[Appendix A](https://docs.arcadedb.com/#Appendix-A)" does not work
* **6. Tools**
  * [x] "6.6 Downgrade to older version of ArcadeDB": `[https://semver.org/,semantic versioning]` seems to be wiki instead of asciidoctor syntax
* **8. SQL**
  * [x] "expand()": "[UNWIND](https://docs.arcadedb.com/SQL-Query.md#unwinding)" is pointing to a non-existing target.
* **11. Apendix**
  *  [x] 11.1 Data Types: "[Managing Dates](http://localhost:8080/#Managing-Dates)" is pointing to a non-existing target.

**Comment by lvca** [comment]
Thanks Christian for the detailed missing parts, I'll fix all of them soon.

**Comment by gramian** [comment]
@lvca I could start fixing some of these linking issues: A recurring issue is that sometimes labels are set but not used, so I would need to know if you prefer to have (custom) labels for sections potentially yielding "nicer" URL fragments, or the auto-generated label from the section title?

**Comment by gramian** [comment]
For the remaining issues in Sections 3,4, and 11 you would have to take a look, but none of those is too urgent I would guess.

**Comment by erickj** [comment]
I think the recent update broke a few links across the SQL commands documentation, e.g. the new [custom label for "SQL Select"](https://github.com/ArcadeData/arcadedb-docs/blame/main/src/main/asciidoc/sql/SQL-Select.adoc#L1) linked from:

https://github.com/erickj/arcadedb-docs/blame/fix-sql-links/src/main/asciidoc/sql/chapter.adoc#L13

I had actually started a change to fix these links myself here, https://github.com/erickj/arcadedb-docs/commit/1557a9bd24a61342c3e4f600c33caa31fd45fafc

but then I discovered that this issue existed, and it was actually moving towards custom labels for the doc sections, I had gone in the opposite direction using the standard adoc labels and realized I had undone some of the recent changes.

Is custom labels the standard for all the SQL keywords & functions? if so I'm happy to contribute another CL fixing up all the tables and internal document links, because at the moment many links are broken through the SQL docs and it makes navigating a pain.

e.g. on https://docs.arcadedb.com/#_sql-truncate-type all the links to "DELETE" and "TRUNCATE  BUCKET" are not right
